source/lex.tex

```diff
@@ -181,7 +181,8 @@
 the members of the basic source character set are intended to
 identify characters from the subset of ISO/IEC 10646 which corresponds to the ASCII
 character set. However, because the mapping from source file characters to the source
-character set (described in translation phase 1) is specified as implementation-defined,
+character set (described in translation phase 1) is specified as
+\impldef{mapping physical source file characters to basic source character set},
```

Comment:
Keep this matching the other occurrence of this phrase if we add "of" or "from" there.